### PR TITLE
optimze & update with keybinds dpemote 

### PR DESCRIPTION
**Pull request by evanillaa**

### CODE REVIEW

**Comment by evanillaa** [comment]
low ms

File: Client/AnimationList.lua [hunk 1]
```diff
@@ -1661,6 +1689,79 @@ DP.PropEmotes = {
        EmoteMoving = true,
    }},
 
+   ["batery"] = {"anim@heists@humane_labs@finale@keycards", "ped_a_enter_loop", "Mechanik - Baterie", AnimationOptions =
```
Comment (phumberdroz) [inline Client/AnimationList.lua:1692]: Would you mind translating this to English? :)

File: Client/AnimationList.lua [hunk 2]
```diff
@@ -1661,6 +1689,79 @@ DP.PropEmotes = {
...
+   {
+       Prop = 'prop_battery_01',
+       PropBone = 18905,
+       PropPlacement = {0.10, -0.15, 0.03, -100.0, 0.0, -10.0},
+       EmoteMoving = true,
+       EmoteLoop = true
+   }},
+   ["plate"] = {"amb@world_human_tourist_map@male@base", "base", "Mechanik - Plate", AnimationOptions =
```
Comment (phumberdroz) [inline Client/AnimationList.lua:1700]: Would you mind translating this to English? :)

File: Client/AnimationList.lua [hunk 3]
```diff
@@ -1661,6 +1689,79 @@ DP.PropEmotes = {
...
+   {
+       Prop = 'p_num_plate_02',
+       PropBone = 28422,
+       PropPlacement = {0.0, 0.0, 0.0, 0.0, 0.0, 0.0},
+       EmoteMoving = true,
+       EmoteLoop = true
+   }},
+   ["bumper"] = {"anim@heists@box_carry@", "idle", "Mechanik - Nárazník", AnimationOptions =
```
Comment (phumberdroz) [inline Client/AnimationList.lua:1708]: Would you mind translating this to English? :)

File: Client/AnimationList.lua [hunk 4]
```diff
@@ -1661,6 +1689,79 @@ DP.PropEmotes = {
...
+   {
+       Prop = "prop_bumper_04",
+       PropBone = 60309,
+       PropPlacement = {0.070, -0.2, 0.255, -175.0, 295.0, 1.0},
+       EmoteLoop = true,
+       EmoteMoving = true,
+   }},
+   ["hood"] = {"anim@heists@box_carry@", "idle", "Mechanik - Kapota", AnimationOptions =
```
Comment (phumberdroz) [inline Client/AnimationList.lua:1716]: Would you mind translating this to English? :)

File: Client/AnimationList.lua [hunk 5]
```diff
@@ -1661,6 +1689,79 @@ DP.PropEmotes = {
...
+   {
+       Prop = "prop_car_bonnet_02",
+       PropBone = 60309,
+       PropPlacement = {0.025, 0.08, 0.255, -145.0, 290.0, 0.0},
+       EmoteLoop = true,
+       EmoteMoving = true,
+   }},
+   ["door"] = {"anim@heists@box_carry@", "idle", "Mechanik - Dvere", AnimationOptions =
```
Comment (phumberdroz) [inline Client/AnimationList.lua:1724]: Would you mind translating this to English? :)

File: Client/AnimationList.lua [hunk 6]
```diff
@@ -1661,6 +1689,79 @@ DP.PropEmotes = {
...
+   {
+       Prop = "prop_car_door_01",
+       PropBone = 60309,
+       PropPlacement = {0.65, -0.25, 0.155, -158.0, 120.0, 0.0},
+       EmoteLoop = true,
+       EmoteMoving = true,
+   }},
+   ["exhaust"] = {"missheistdocksprep1hold_cellphone", "static", "Mechanik - Výfuk", AnimationOptions =
```
Comment (phumberdroz) [inline Client/AnimationList.lua:1732]: Would you mind translating this to English? :)

File: Client/AnimationList.lua [hunk 7]
```diff
@@ -1661,6 +1689,79 @@ DP.PropEmotes = {
...
+   {
+       Prop = "prop_car_exhaust_01",
+       PropBone = 57005,
+       PropPlacement = {0.25, -0.1, 0.0, 0.0, 266.0, -40.0},
+       EmoteLoop = true,
+       EmoteMoving = true,
+   }},
+   ["carseat"] = {"anim@heists@box_carry@", "idle", "Mechanik - Sedacka", AnimationOptions =
```
Comment (phumberdroz) [inline Client/AnimationList.lua:1740]: Would you mind translating this to English? :)

File: Client/AnimationList.lua [hunk 8]
```diff
@@ -1661,6 +1689,79 @@ DP.PropEmotes = {
...
+   {
+       Prop = "prop_car_seat",
+       PropBone = 60309,
+       PropPlacement = {0.05, -0.1, 0.255, -145.0, 150.0, 1.5},
+       EmoteLoop = true,
+       EmoteMoving = true,
+   }},
+   ["wheel"] = {"anim@heists@box_carry@", "idle", "Mechanik - Kolo", AnimationOptions =
```
Comment (phumberdroz) [inline Client/AnimationList.lua:1748]: Would you mind translating this to English? :)

File: Client/AnimationList.lua [hunk 9]
```diff
@@ -1661,6 +1689,79 @@ DP.PropEmotes = {
...
+   {
+       Prop = "prop_wheel_03",
+       PropBone = 60309,
+       PropPlacement = {0.016, 0.15, 0.255, -185.0, 250.0, 0.0},
+       EmoteLoop = true,
+       EmoteMoving = true,
+   }},
+   ["engine"] = {"anim@heists@box_carry@", "idle", "Mechanik - Motor", AnimationOptions =
```
Comment (phumberdroz) [inline Client/AnimationList.lua:1756]: Would you mind translating this to English? :)

File: Client/AnimationList.lua [hunk 10]
```diff
@@ -1718,7 +1819,7 @@ DP.PropEmotes = {
        PropPlacement = {0.0, 0.0, 0.0, 0.0, 0.0, 0.0},
        EmoteLoop = true,
        EmoteMoving = true,
-   }},
+   }},
```
Comment (phumberdroz) [inline Client/AnimationList.lua:1822]:
```suggestion
   }},
```

File: Client/Emote.lua [hunk 11]
```diff
@@ -203,13 +140,10 @@ function EmoteCancel()
...
     IsInAnimation = false
   end
-
-  -- Added
-
   if SmokingWeed then
     SmokingWeed = false
     RelieveCount = 0
-  end
+  end
```
Comment (phumberdroz) [inline Client/Emote.lua:146]:
```suggestion
  end
```

File: Client/Emote.lua [hunk 12]
```diff
@@ -386,26 +331,28 @@ function CheckGender()
     PlayerGender = "female"
   end
   DebugPrint("Set gender as = ("..PlayerGender..")")
-end
+end]]
```
Comment (phumberdroz) [inline Client/Emote.lua:334]: You could consider removing the entire function

**Review by phumberdroz — CHANGES_REQUESTED** [review]
Generally there is a couple of things that could be improved. (see my comments)

Also it would be nice to get some annotation in regards to your motivation to do a certain change.

Overall I am uncertain about adding your name to the function headers as git keeps track of those changes, but that is up to the Project Maintainers to decide.

**Review by evanillaa — COMMENTED** [review]
y

**Pull request closed without merging**